db/migrate/20170404033714_create_phrases.rb

```diff
@@ -1,9 +0,0 @@
-class CreatePhrases < ActiveRecord::Migration[5.0]
```

Comment:
Instead of deleting the old migration that creates the phrases, would you consider making a new migration that deletes the phrases table?